x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -447,22 +455,25 @@ export const visualizationTypes: VisualizationType[] = [
       defaultMessage: 'H. Bar',
     }),
     fullLabel: i18n.translate('xpack.lens.xyVisualization.barHorizontalFullLabel', {
-      defaultMessage: 'Horizontal bar',
+      defaultMessage: 'Bar horizontal',
     }),
+    groupLabel: groupLabelForBar,
   },
   {
     id: 'bar_stacked',
     icon: LensIconChartBarStacked,
     label: i18n.translate('xpack.lens.xyVisualization.stackedBarLabel', {
-      defaultMessage: 'Stacked bar',
+      defaultMessage: 'Bar stacked',
```

Comment:
To be consistent with naming and match the designs.

```suggestion
      defaultMessage: 'Bar vertical stacked',
```